RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3124 
Task addr: 2147522956 
Task slot: 0 
Task slot TaskAddr: 2147522956 
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 1  from app 1 
taskSlot run: 0 
SendingQueue_front: 1 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
Task P1 INIT
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 3 
API_Try2Send success!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 7 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 2021055 
TotalEnergy: 23297938 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 977781 
TotalEnergy: 45472089 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1000001 
TotalEnergy: 46349954 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 999900 
TotalEnergy: 46584286 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1001415 
TotalEnergy: 46648787 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1020706 
TotalEnergy: 47621076 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 977719 
TotalEnergy: 45706155 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 999939 
TotalEnergy: 46582675 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1000073 
TotalEnergy: 46583594 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 999799 
TotalEnergy: 46343467 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1000001 
TotalEnergy: 46355204 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1001454 
TotalEnergy: 46651516 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1010707 
TotalEnergy: 47233126 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 987679 
TotalEnergy: 46089297 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1000040 
TotalEnergy: 46583527 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 999900 
TotalEnergy: 46582366 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1000174 
TotalEnergy: 46598366 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 1003233 
TotalEnergy: 46726394 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
dynamicEnergy_Router >> 6359375 
total instructions~~~> 997779 
TotalEnergy: 46480852 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 101909 
TotalEnergy: 10627532 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280008 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280918 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280008 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280932 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280027 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280932 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 6359375 
total instructions~~~> 99990 
TotalEnergy: 10280027 
TX interruption catched
API_Try2Send failed - empty SendQueue!
